src/ResourceManager/KeyVault/documentation/upcoming-breaking-changes.md

```diff
@@ -27,6 +27,90 @@
 
 # Upcoming Breaking Changes
 
+The following cmdlets were affected this release:
+
+**Add-AzureKeyVaultCertificate**
+- The -Certificate parameter has become mandatory.
+
+**Set-AzureKeyVaultManagedStorageSasDefinition**
+- The -Parameter parameter has become optional and no longer has a position.
```

Comment:
@maddieclayton we shouldn't need to call out the parameter is now optional, just the positional change